packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableClaimableRow.tsx

```diff
@@ -32,35 +32,48 @@ type CommonProps = {
 }
 
 function ClaimableRowStatus({ tx }: CommonProps) {
-  const { parentLayer, layer } = useChainLayers()
+  const { parentLayer, layer: childLayer } = useChainLayers()
   const matchingL1TxId = tx.isCctp
     ? tx.cctpData?.receiveMessageTransactionHash
     : getWithdrawalClaimParentChainTxDetails(tx)?.txId
 
+  const sourceLayer = tx.isWithdrawal ? childLayer : parentLayer
+  const destLayer = tx.isWithdrawal ? parentLayer : childLayer
```

Comment:
NIT: Tbh I'd prefer having full `destinationLayer` here, as we're using the same in the rest of the project.

Reply:
updated